app/views/forever_style_guide/sections/components/_modals.erb

```diff
@@ -0,0 +1,119 @@
+<!-- Trigger the modal with a button -->
+<div class='row'>
+  <div class='col-md-6'>
+    <h3>Decision/Info:</h3>
+    <button type="button" class="btn btn-danger btn-lg" data-toggle="modal" data-target="#info">Delete Photo</button>
+  </div>
+  <div class='col-md-6'>
+    <h3>Product:</h3>
+    <button type="button" class="btn btn-primary btn-lg" data-toggle="modal" data-target="#product">Add to Cart</button>
+  </div>
+  <div class='col-md-6'>
+    <h3>Media:</h3>
+    <button type="button" class="btn btn-artisan btn-lg" data-toggle="modal" data-target="#media">Learn More</button>
+  </div>
+  <div class='col-md-6'>
+    <h3>Field:</h3>
+    <button type="button" class="btn btn-services btn-lg" data-toggle="modal" data-target="#field">View Promotion</button>
+  </div>
+</div>
+
+<!-- Decision Modals -->
+<div id="info" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header modal-theme color_block-danger">
+        <span type="button" class="modal-close-position" data-dismiss="modal"><i class="fa fa-close fa-2x"></i></span>
+        <h3 class="modal-title"><i class="fa fa-trash-o"></i> Delete Photo?</h3>
+      </div>
+      <div class="modal-body modal-body-info">
+        <p>Are you sure you want to <strong>delete this photo from all locations</strong>, including inbox, albums, tags and dates?</p>
+      </div>
+      <div class="modal-footer modal-footer-info">
+        <span class="pull-left modal-footer-txt"><i>This action cannot be undone.</i></span>
+        <button type="button" class="btn btn-link" data-dismiss="modal">Cancel</button>
+        <button type="button" class="btn btn-primary" data-dismiss="modal">Yes, Delete</button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- Product Modals -->
+<div id="product" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header modal-theme color_block-primary">
+        <span type="button" class="modal-close-position" data-dismiss="modal"><i class="fa fa-close fa-2x"></i></span>
+        <h3 class="modal-title"><i class="fa fa-check-circle-o"></i> 1 Item Added to Cart</h3>
```

Comment:
Will the modal title and icon always be white? Could we just use a color-white class here?